### ISSUE TITLE
[arm32/Ubuntu] Failure in R2R JitStress=1 job

### ISSUE THREAD

**Issue by BruceForstall** [dotnet/coreclr#17605]
https://ci.dot.net/job/dotnet_coreclr/job/master/view/arm/job/jitstress/job/arm_cross_checked_ubuntu_r2r_jitstress1_flow/

It appears to fail with a hang in this test:
```
FAILED   - JIT/Regression/JitBlue/DevDiv_255294/DevDiv_255294/DevDiv_255294.sh
               BEGIN EXECUTION
               in takeLock
               /ssd/j/workspace/dotnet_coreclr/master/jitstress/arm_cross_checked_ubuntu_r2r_jitstress1_tst/bin/tests/Linux.arm.Checked/Tests/Core_Root/crossgen /Platform_Assemblies_Paths /ssd/j/workspace/dotnet_coreclr/master/jitstress/arm_cross_checked_ubuntu_r2r_jitstress1_tst/bin/tests/Linux.arm.Checked/Tests/Core_Root:/ssd/j/workspace/dotnet_coreclr/master/jitstress/arm_cross_checked_ubuntu_r2r_jitstress1_tst/bin/tests/Linux.arm.Checked/JIT/Regression/JitBlue/DevDiv_255294/DevDiv_255294/IL:/ssd/j/workspace/dotnet_coreclr/master/jitstress/arm_cross_checked_ubuntu_r2r_jitstress1_tst/bin/tests/Linux.arm.Checked/JIT/Regression/JitBlue/DevDiv_255294/DevDiv_255294 /in DevDiv_255294.org /out DevDiv_255294.exe
               ./DevDiv_255294.sh: line 156: 17678 Terminated              $_DebuggerFullPath "$CORE_ROOT/crossgen" /Platform_Assemblies_Paths $CORE_ROOT:$PWD/IL:$PWD /in DevDiv_255294.org /out DevDiv_255294.exe
               Crossgen failed with exitcode: 143
               in ReleaseLock
```

This is consistent in multiple runs, e.g.:

https://ci.dot.net/job/dotnet_coreclr/job/master/job/jitstress/job/arm_cross_checked_ubuntu_r2r_jitstress1_tst/3/consoleText

**Comment by BruceForstall** [comment]
@dotnet/arm32-contrib

**Comment by BruceForstall** [comment]
Also with JitStress=2:

https://ci.dot.net/job/dotnet_coreclr/job/master/view/arm/job/jitstress/job/arm_cross_checked_ubuntu_r2r_jitstress2_tst/3/consoleText

**Comment by BruceForstall** [comment]
If it's a known issue, we need to disable this test (for R2R JitStress) so the job doesn't time out.

**Comment by sandreenko** [comment]
It is the same issue, the fix #16663 won't be in 2.1.
> If it's a known issue, we need to disable this test (for R2R JitStress) so the job doesn't time out.

The issue exists on all platforms (for am64, x86 only with JitStress=2), I believe we do not have mechanism to disable one test in stress or R2R mode on all platforms, what is the best way to do it?

**Comment by BruceForstall** [comment]
> The issue exists on all platforms

Does the job time out for all platforms?

> what is the best way to do it?

Maybe the generated cmd/sh file can check for the R2R + JitStress flags and simply return?

Or use (and maybe invent) a mechanism to disable the test for any JitStress mode?

**Comment by sandreenko** [comment]
> Does the job time out for all platforms?

Yes, it takes 5 hours on amd64 with r1r and jitstress=1, [example amd64](https://ci.dot.net/job/dotnet_coreclr/job/master/job/jitstress/job/x64_checked_windows_nt_r2r_jitstress1/lastFailedBuild/testReport/), [example arm](https://ci.dot.net/job/dotnet_coreclr/job/master/job/jitstress/job/arm_cross_checked_windows_nt_r2r_jitstress1_tst_prtest/26/artifact/bin/tests/Windows_NT.arm.Checked/Smarty.run.0/Smarty.0.fail.smrt/*view*/). It is described in #16573.
